### PR TITLE
Add documentation on converting from WES 1.0 swagger 2.0 to OpenAPI 3.0

### PR DESCRIPTION
**Pull request by wshands**
Add documentation on converting from WES 1.0 swagger 2.0 to OpenAPI 3.0
DOCK-573

### CODE REVIEW

File: swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+##How WES OpenApi 3.0 yaml was created from Swagger 2.0 yaml
+
+SwaggerHub was used to convert the WES 1.0.0 Swagger 2.0 yaml to OpenAPI 3.0 yaml
+
+####Steps to convert:
```
Comment (garyluu) [inline swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md]: Headings need space after the `#`

File: swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+1. Launch SwaggerHub https://app.swaggerhub.com/home
+2. Create an account and login to SwaggerHub
+3. Click on the 'plus' (**+**) icon on the left side of the page
+3. Select 'Import and Document API'
```
Comment (garyluu) [inline swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md]: Simpler to just put `1` all the way down

File: swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+4. In the Import API dialog paste the WES Swagger 2.0 URL: 
+https://raw.githubusercontent.com/ga4gh/workflow-execution-service-schemas/1.0.0/openapi/workflow_execution_service.swagger.yaml into the
+ 'Path or URL' edit box
+5. Click 'Import'
+..You should now be able to see the The WES Swagger 2.0 yaml in the editor
+6. Click on the green box with the down arrow at the top right of the editor
+7. Select 'Convert to OpenAPI 3.0'
+8. In the 'Convert to OpenAPI 3.0?' dialog click 'Convert & Update'
+9. Click the 'Export^' button on the uppper right hand area of the page
```
Comment (garyluu) [inline swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md]: The ^ should be pointed down, but calling it the Export dropdown menu might be sufficient.

Comment (garyluu) [reply swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md]: image

**Comment by denis-yuen** [comment]
circle ci is false positive

**Comment by garyluu** [comment]
IMO, you can [skip ci] for the next commit

File: swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md (hunk 1 above)
Comment (wshands) [reply swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md]: done

File: swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md (hunk 2 above)
Comment (wshands) [reply swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md]: fixed

File: swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md (hunk 3 above)
Comment (wshands) [reply swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md]: fixed

File: swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md (hunk 2 above)
Comment (wshands) [reply swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md]: done

File: swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+## How WES OpenApi 3.0 yaml was created from Swagger 2.0 yaml
+
+SwaggerHub was used to convert the WES 1.0.0 Swagger 2.0 yaml to OpenAPI 3.0 yaml
+
+#### Steps to convert:
+1. Launch SwaggerHub https://app.swaggerhub.com/home
```
Comment (coverbeck) [inline swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md]: I would say `Navigate to SwaggerHub`, but don't feel too strongly about it.

**Pull request merged**